tests/components/apprise/test_notify.py

```diff
@@ -0,0 +1,68 @@
+"""The tests for the apprise notification platform."""
+from unittest.mock import patch
+
+from homeassistant.setup import async_setup_component
+
+BASE_COMPONENT = "notify"
+
+
+async def test_apprise_config_load_fail(hass):
+    """Test apprise configuration failures."""
+
+    config = {"apprise": {"name": "test", "platform": "apprise", "config": "/path/"}}
+
+    with patch("apprise.AppriseConfig.add", return_value=False):
```

Comment:
Be careful to patch in the correct place.
https://docs.python.org/3/library/unittest.mock.html#where-to-patch

I think we should change this to:
```suggestion
    with patch("homeassistant.components.apprise.notify.AppriseConfig.add", return_value=False):
```